content/en/hosting/cht/migration/preparing-for-5.md

```diff
@@ -162,5 +162,15 @@ Nouveau has the following impact on all CHT deployments upgrading to 5.0:
 * The Nouveau index data on the server will be stored in `${COUCHDB_DATA}/nouveau` for single-node CouchDBs and in `${DB1_DATA}/nouveau` for clustered CouchDBs.
 * The following `medic-client` views no longer exist. Be sure to update any custom scripts which use them:  `contacts_by_freetext`,  `contacts_by_type_freetext` and  `reports_by_freetext` .
 
+### Replication and online search don't work after upgrade
```

Comment:
maybe we should include "temporarily" here, lots of people don't end up reading the fine print.